<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QuickMsg - &lt;Cnonim&gt;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>350</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::East</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>how to</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_2">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string>Хостинг предоставлен 
            Литвиновым Дмитрием Николаевичем

Порядок работы с программой:

1.Выполнить вход*.(operation-&gt;enter to network)
2.Общаться**.
3.Выполнить выход***.(operation-&gt;exit)

* При необходимости смены своего имени выбирете &quot;operation-&gt;change nick&quot;.
** Для создания нового диалога выбирете &quot;operation-&gt;online&quot;, после чего появиться окно со списком пользователей, который активны в данный момент.
Выбирете двойным щелчком собеседника, после чего появится новая вкладка с диалогом с именем собеседника.
Так же перед выбором собеседника можно установить метод шифрования(выпадающее меню) и ключ для шифрования(текстовое поле под меню).
***Каждый корректный выход продлевает срок службы Вашего компьютера на одну неделю!!!</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>70</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="text">
         <string>SEND</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>1</width>
        <height>1</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperation">
    <property name="title">
     <string>operation</string>
    </property>
    <addaction name="actionEnter"/>
    <addaction name="actionChange_nickname"/>
    <addaction name="actionOnline"/>
    <addaction name="separator"/>
    <addaction name="actionShow_encoded_message"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>about</string>
    </property>
    <addaction name="actionAbout_program"/>
    <addaction name="actionAbout_authors"/>
   </widget>
   <addaction name="menuOperation"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionChange_nickname">
   <property name="text">
    <string>change nickname</string>
   </property>
  </action>
  <action name="actionOnline">
   <property name="text">
    <string>online</string>
   </property>
  </action>
  <action name="actionEnter">
   <property name="text">
    <string>enter to network</string>
   </property>
  </action>
  <action name="actionAbout_program">
   <property name="text">
    <string>about program</string>
   </property>
  </action>
  <action name="actionAbout_authors">
   <property name="text">
    <string>about authors</string>
   </property>
  </action>
  <action name="actionShow_encoded_message">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>show encoded message</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
